torch/optim/lr_scheduler.py

```diff
@@ -4,6 +4,8 @@
 from torch._six import inf
 from bisect import bisect_right
 from functools import partial
+
+import numpy as np
```

Comment:
numpy is an optional dependency into PyTorch.
Please remove the dependence of np in this file.